packages/types-algosdk/types/index.d.ts

```diff
@@ -56,7 +57,27 @@ declare module 'algosdk' {
 
     function AssetDef(...args: any[]): void;
 
-    function DryrunRequest(...args: any[]): void;
+    class DryrunRequest {
```

Comment:
do we have this exported in algosdk ?

Reply:
yes, in `modelsv2` class [https://github.com/algorand/js-algorand-sdk/blob/develop/src/main.ts#L251](https://github.com/algorand/js-algorand-sdk/blob/develop/src/main.ts#L251)